bin/test_env.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
```

Comment:
This comes from Socorro. It gets around the issue where Circle CI can run docker-compose, but services can't mount volumes. We have to create a data container and copy the data into it and then mount that all without docker-compose.

Because I like running the tests in the local development environment in the same way that they're run in Circle CI, we do this mildly complicated song-and-dance. Further, I like having a test shell environment, so this enables that, too.